docs/nodes/light-node.mdx

```diff
@@ -708,7 +708,7 @@ is usually exposed on port 9090):
 <TabItem value="arabica" label="Arabica">
 
 ```sh
-celestia light start --core.ip <ip-address> --core.grpc.port <port> --gateway --gateway.addr <ip-address> --gateway.port <port> --p2p.network arabica
+celestia light start --core.ip <ip-address> --core.grpc.port <port> --gateway --gateway.addr <ip-address> --gateway.port <port> --p2p.network arabica-2
```

Comment:
```suggestion
celestia light start --core.ip <ip-address> --core.grpc.port <port> --gateway --gateway.addr <ip-address> --gateway.port <port> --p2p.network <network>
```

Reply:
although this is an arabica command, for sake of formatting, will keep same as rest of command. i.e. `<network>` instead of `arabica` or `arabica-2`